module/x/gravity/types/ethereum.go

```diff
@@ -2,7 +2,9 @@ package types
 
 import (
 	"bytes"
+	"encoding/hex"
 	"fmt"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	"strings"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
```

Comment:
Do we have any convention to group imports? If not this suggestion can be ignored.
```suggestion
	"strings"

	sdk "github.com/cosmos/cosmos-sdk/types"
	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
```